core/common/src/test/java/alluxio/ConfigurationTestUtils.java

```diff
@@ -140,6 +140,9 @@ public static InstancedConfiguration defaults() {
     conf.put(PropertyKey.MASTER_PERSISTENCE_INITIAL_INTERVAL_MS, "20ms");
     conf.put(PropertyKey.MASTER_PERSISTENCE_SCHEDULER_INTERVAL_MS, "20ms");
 
+    // faster refresh
+    conf.put(PropertyKey.MASTER_WORKER_INFO_CACHE_REFRESH_TIME, "100ms");
```

Comment:
should all integration tests just use a shorter refresh time, like 20ms?

Reply:
I am worried if the too frequent getWorkerInfoList calls will create locking contentions in integration tests.